pipeline_tools/shared/submission/parse_cromwell_metadata.py

```diff
@@ -13,9 +13,18 @@ def parse_optimus_metadata(metadata_json):
 def parse_SS2_metadata(metadata_json):
     with open(metadata_json, 'r') as f:
         metadata = json.load(f)
+
+    # find reference fasta path in metadata.json
     ref_fasta_path = metadata['inputs']['genome_ref_fasta']
+
+    # find pipeline version in metadata.json
+    # project level run should be the version of the MultiSampleSmartSeq2 pipeline
     pipeline_version = metadata['calls']['MultiSampleSmartSeq2.AggregateLoom'][0]['inputs']['pipeline_version']
```

Comment:
```suggestion
    multi_sample_pipeline_version = metadata['calls']['MultiSampleSmartSeq2.AggregateLoom'][0]['inputs']['pipeline_version']
```

Reply:
ah i did this in the wdl but forgot to change it here, good catch!